errors/es-mx.po

```diff
@@ -0,0 +1,1474 @@
+msgstr "Esto significa:"
+
+#: templates/ERR_FTP_NOT_FOUND+html.body.div.p:34-1
+msgid ""
+"This might be caused by an FTP URL with an absolute path (which does not "
+"comply with RFC 1738). If this is the cause, then the file can be found at "
+"<a href=\"%B\">%B</a>."
+msgstr ""
+"Esto podria ser causado por una URL del FTP con ruta absoluta (que no cumple "
+"con RFC 1738). Si esta es la causa, entonces el archivo puede ser encontrado "
+"en <a href=\"%B\">%B</a>."
+
+#: templates/ERR_SECURE_CONNECT_FAIL+html.body.div.p:35-1
+msgid ""
+"This proxy and the remote host failed to negotiate a mutually acceptable "
+"security settings for handling your request. It is possible that the remote "
+"host does not support secure connections, or the proxy is not satisfied with "
+"the host security credentials."
+msgstr ""
+"Este proxy y el host remoto no han podido negociar una confguración de seguridad "
+"mutuamente aceptable para manejar su solicitud. Es posible que el host remoto sea "
+"incompatible con conexiones seguras o el proxy no esté satisfecho con las "
+"credenciales de seguridad del host."
+
+#: templates/ERR_ACL_TIME_QUOTA_EXCEEDED+html.body.div.p:27-1
+msgid ""
+"This proxy limits your time online with a quota. Your time budget is now "
+"empty but will be refilled when the configured time period starts again."
+msgstr ""
+"Este proxy limita el tiempo en línea mediante cuotas. El tiempo asingnado "
+"se ha agotado pero se reiniciara cuando el periodo configurado inicie "
+"nuevamente."
+
+
+#: templates/ERR_CANNOT_FORWARD+html.body.div.p:27-1
+msgid ""
+"This request could not be forwarded to the origin server or to any parent "
+"caches."
+msgstr ""
+"Esta solicitud no se pudo reenviar al servidor de origen ni a ningún caché "
+"principal."
+
+#: templates/ERR_ACL_TIME_QUOTA_EXCEEDED+html.body.div.blockquote.p:24-1
+msgid "Time Quota Exceeded."
+msgstr "Cuota de tiempo excedida."
+
+#: templates/ERR_AGENT_CONFIGURE+html.body.div.div.ul.li:38-1
+#: templates/ERR_AGENT_WPAD+html.body.div.div.ul.li:38-1
+msgid ""
+"Tools -&gt; Internet Options -&gt; Connection -&gt; LAN Settings -&gt;Proxy"
+msgstr ""
+"Herramientas -&gt; Opciones de Internet  -&gt; Conexión -&gt; Preferencias "
+"de RED -&gt;Proxy"
+
+#: templates/ERR_AGENT_CONFIGURE+html.body.div.div.ul.li:30-1
+#: templates/ERR_AGENT_WPAD+html.body.div.div.ul.li:30-1
+msgid ""
+"Tools -&gt; Options -&gt; Advanced -&gt; Network -&gt; Connection Settings"
+msgstr ""
+"Herramientas -&gt; Opciones -&gt; Avanzado -&gt; Red -&gt; Ajustes de "
+"conexión"
+
+#: templates/ERR_AGENT_CONFIGURE+html.body.div.div.ul.li:46-1
+#: templates/ERR_AGENT_WPAD+html.body.div.div.ul.li:46-1
+msgid ""
+"Tools -&gt; Preferences -&gt; Advanced -&gt; Network -&gt; Proxy Servers"
+msgstr ""
+"Herramientas -&gt; Opciones -&gt; Avanzado -&gt; Red -&gt; Servidores Proxy"
+
+#: templates/ERR_CONFLICT_HOST+html.body.div.blockquote.pre:24-1
+msgid "URI Host Conflict"
+msgstr "Conflicto en el host URI"
```

Comment:
This is a page title/header, thus the upper casing in English.  The meaning of this text is supposed to be that the host name in the URI is conflicting with something (usually the HTTP Host header or TLS SNI). Is this translation conveying that meaning still?

Reply:
Yes, it does.